### ISSUE TITLE
Reduced Motion accessibility option

### ISSUE THREAD

**Issue by immortius** [Dimension-Delvers/dimension-delvers-mod#96]
Consider vanilla effects that should be addressed by the Reduced Motion accessibility option, which is intended to disable or reduce UI animation/camera shake/screen effects. I note that graphics enhancement mods option include a reduce motion and/or reduce fov effects option.
